src/main/java/step4/model/Cars.java

```diff
@@ -0,0 +1,42 @@
+package step4.model;
+
+import step4.strategy.CarMovingStrategy;
+import step4.strategy.Strategy;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+  private final List<Car> cars;
+
+  private final Strategy strategy;
+
+  public Cars(List<Car> cars) {
+    this.cars = cars;
+    this.strategy = new CarMovingStrategy();
+  }
+
+  public List<Car> getCars() {
+    return cars;
+  }
+
+  public boolean runCycle() {
+    cars.forEach(car -> car.move(strategy));
+    return true;
+  }
+
+  public List<String> calcChampions(){
+
+    int maxValue =getMaxLocation();
+    if(maxValue==0) return new ArrayList<>();
+    List<String> champs;
+    champs = cars.stream().filter(car->car.showCarLocation()==maxValue).map(Car::showCarName).collect(Collectors.toList());
+    return champs;
+  }
```

Comment:
```suggestion
  public List<String> calcChampions(){
    int maxValue = getMaxLocation();
    if (maxValue == 0) {
        return Collections.emptyList();
    }
    
    return cars.stream()
        .filter(car->car.showCarLocation() == maxValue)
        .map(Car::showCarName)
        .collect(Collectors.toList());
  }
```
가독성을 조금만 더 신경쓰면 좋을 것 같아요!